rasa/utils/endpoints.py

```diff
@@ -226,3 +226,23 @@ def float_arg(
     except (ValueError, TypeError):
         logger.warning(f"Failed to convert '{arg}' to float.")
         return default
+
+
+def int_arg(
+    request: Request, key: Text, default: Optional[int] = None
+) -> Optional[int]:
+    """Return a passed argument cast as an int or None.
```

Comment:
Can you please also describe the other paramaters and then return values using `Args:` and `Returns`:?
```suggestion
    """Returns a passed argument cast as an int or None.
```

Reply:
I've added for `bool_arg`, `float_arg` and `int_arg`

Reply:
Awesome. Minor comment: Could you please change `Return` to `Returns`? We try to use descriptive docstrings in the codebase.

Reply:
Sure, I forgot to update this one